### DISCUSSION TITLE
should the absolute value of a number be printed with a minus sign in front of it?

### DISCUSSION THREAD

**Discussion by brandy125** [gnudatalanguage/gdl#1679] [General]
IDL and GDL behave the same here but is it correct?
Shouldn't the printout of the absolute value of a number get a positive sign in front of it?

This is ok for me:

```
GDL> print,-0.0
     -0.00000
```

...but this?

```
GDL> print,abs(-0.0)
     -0.00000
```

**Comment by alaingdl** [comment]
I would say we are wrong :smiley: 

```
GDL (1.0.3git)> neg=-1*(0.) & print, neg, abs(neg) , sin(neg)
     -0.00000     -0.00000     -0.00000

IDL (8.8.0)>neg=-1*(0.) & print, neg, abs(neg) , sin(neg)
     -0.00000      0.00000     -0.00000

```


(yes, -0.0 & +0.0 do exist ... IEEE 754 https://en.wikipedia.org/wiki/Signed_zero)

**Comment by GillesDuvert** [comment]
So, this may be an issue 😀
